LAYOUT_STYLE_GUIDE.md

```diff
@@ -399,3 +400,7 @@ Finally, the **HTML (Recommended)** code option must be the one that is copy + p
 When a user adds a Codepen embed to a lesson, a maintainer should fork the embed to the official [TOP Codepen](https://codepen.io/TheOdinProjectExamples/) account. When necessary, the name of new pens should be updated to better reflect their purpose, e.g. `Simple SVG Example` for a pen showing a simple SVG or `max-width | CSS Responsiveness` for a pen about the `max-width` property.
 
 After forking a pen to the TOP account and ensuring the embeds options from above are selected, the lesson the original embed is from should be updated to include the forked, TOP version instead.
+
+## English
+
+As a general note, TOP follows American English and American style punctuation.
```

Comment:
Small addition that isn't exactly super important, so I'm also fine leaving this line as-is.

```suggestion
As a general note, TOP follows American English and American style punctuation. When adding content to the curriculum, be sure to follow this practice for consistency across lessons.
```

Reply:
Great! I originally had another sentence about consistency in the curriculum, but I took it out since it wasn't part of my original proposal for this note. I think it's a helpful point to make on "why" we have this guideline so let's put it in.